schema.py

```diff
@@ -0,0 +1,810 @@
+#!/usr/bin/env python3
+
+# -*- coding: utf-8 -*-
+# Copyright (C) 2019 NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""GraphQL API schema via Graphene implementation."""
+
+import graphene
```

Comment:
Yeah I've been tossing this one up.. All the docs have examples with `graphene` imported, so I thought I'd play it safe. However, given that I've isolated the schema from the server somewhat, it might make sense (definitely would de-clutter it)..

Reply:
Done.